sphinx_docs/source/nersc-compilers.rst

```diff
@@ -100,3 +100,79 @@ When running MAESTROeX, we seem to need::
 
 otherwise we get an ``Erroneous arithmetic error``.
 
+
+
+Cori GPU
+--------
+
+To use the Cori GPU system, you first need to load the ``cgpu`` module::
+
+  module load cgpu
```

Comment:
I prefer `module purge` and then `module load cgpu` because most of the modules make no sense outside the confines of `cgpu`.

Reply:
ah, thanks! done :)